primitives/account/src/staking_contract/traits.rs

```diff
@@ -660,6 +714,15 @@ impl AccountInherentInteraction for StakingContract {
                             .current_lost_rewards
                             .remove(slot.slot as usize);
                     }
+                    logs.insert(
```

Comment:
Does the order of logs matter here or can this also be
```suggestion
                    logs.push(
```
?

If the order does matter, add a comment explaining why.